schema/proto3/common.proto

```diff
@@ -35,14 +37,9 @@ enum OperationType {
     GET = 2;
     DELETE = 3;
     INFO = 4;
-    NEIGHBOR_CREATE_UPDATE = 5;
-    NEIGHBOR_GET = 6;
-    NEIGHBOR_DELETE = 7;
-    CREATE_UPDATE_GATEWAY = 8;
-
-    FINALIZE = 9;              // For Mizar only
-    CREATE_UPDATE_SWITCH = 10; // For Mizar only
-    CREATE_UPDATE_ROUTER = 11; // For Mizar only
+    NEIGHBOR_CREATE_UPDATE = 5; // To be removed after using NeighborState
```

Comment:
if we put neighbor into port as field, then we do NOT need NEIGHBORXXX type and logic to handle that